test/common/testlib.py

```diff
@@ -93,7 +93,7 @@ def __init__(self, address, label, port=None, headless=True):
             self.port = port
         self.default_user = "admin"
         self.label = label
-        self.cdp = cdp.CDP("C.utf8", headless, trace=opts.trace)
+        self.cdp = cdp.CDP("C.utf8", headless, verbose=opts.trace, trace=opts.trace)
```

Comment:
Oops, this should say opts.verbose.

Reply:
I actually started with that, but testlib.py actually doesn't have an `opts.verbose`, only `trace`. But I think only showing the browser startup/shutdown logs with `-t` seems good enough..

Reply:
Makes sense.

Reply:
We do the same in the `testvm.Machine` instantiation, BTW:

    machine = machine_class(verbose=opts.trace, networking=networking, image=image, **kwargs)